### PR TITLE
Aimbot

### PR DESCRIPTION
**Pull request by ManiacMoustache**
Relies on DoofConfigUpdate

### CODE REVIEW

File: .deploy_cfg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[auth]
+hostname = 10.32.0.2
```
Comment (tuffinmuffin) [inline .deploy_cfg]: remove file?

File: .installer_config [hunk 2]
```diff
@@ -0,0 +1,3 @@
+[auth]
+hostname = 3200
```
Comment (tuffinmuffin) [inline .installer_config]: remove file?

File: deploy_utils/.installer_config [hunk 3]
```diff
@@ -0,0 +1,3 @@
+[auth]
+hostname = 3200
+
```
Comment (tuffinmuffin) [inline deploy_utils/.installer_config]: delete?

File: deploy_utils/Installer-script-win/.installer_config [hunk 4]
```diff
@@ -0,0 +1,3 @@
+[auth]
+hostname = 3200
```
Comment (tuffinmuffin) [inline deploy_utils/Installer-script-win/.installer_config]: delete?

**Review by tuffinmuffin — DISMISSED** [review]

File: components/shooterLogic.py [hunk 5]
```diff
@@ -1,7 +1,7 @@
 from robotMap import XboxMap
+import logging as log
```
Comment (ChrisHirsch) [inline components/shooterLogic.py:2]: Needed?

**Review by ChrisHirsch — COMMENTED** [review]
Clean up some code and lets take this branch out for a drive and we're good to go

File: components/shooterLogic.py (hunk 5 above)
Comment (ManiacMoustache) [reply components/shooterLogic.py:2]: Yes, we use it multiple times throughout the file

**Review by ChrisHirsch — APPROVED** [review]
+1 ok lets see how this behaves...there could definitely be some improvements but overall I'm liking the documentation, better comments and the components

**Pull request merged**